frontend/src/modules/Worksheets/views/WorksheetView.js

```diff
@@ -47,8 +43,32 @@ import {
 import {
   SQLQueryEditor,
   WorksheetEditFormModal,
-  WorksheetResult
+  WorksheetResult,
+  WorksheetTextToSQLEditor,
+  WorksheetDocAnalyzer,
+  WorksheetSQLEditor
 } from '../components';
+import { isFeatureEnabled } from 'utils';
+
+const tabs = [
+  {
+    label: 'SQL Editor',
+    value: 'SQLEditor',
+    active: true
+  },
+  {
```

Comment:
It looks a bit ugly in the FE because instead of camelcase all letters are uppercase
![image](https://github.com/user-attachments/assets/ee41635d-3008-40c5-a694-47699d19edb9)

Reply:
Changing to `Text-To-SQL Editor`
![Screenshot 2024-10-24 at 9 26 14 AM](https://github.com/user-attachments/assets/f7504bf9-75ba-42ab-8eba-d2b3fa4a4c76)